src/Analyzers/CSharp/Analyzers/NewLines/ConsecutiveBracePlacement/CSharpConsecutiveBracePlacementDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.NewLines.ConsecutiveBracePlacement
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal sealed class CSharpConsecutiveBracePlacementDiagnosticAnalyzer : AbstractBuiltInCodeStyleDiagnosticAnalyzer
+    {
+        public CSharpConsecutiveBracePlacementDiagnosticAnalyzer()
+            : base(IDEDiagnosticIds.ConsecutiveBracePlacementDiagnosticId,
+                   EnforceOnBuild.WhenExplicitlyEnabled,
```

Comment:
I think EnforceOnBuildValues should be used instead.

Reply:
done.